lib/Doctrine/Locator/Injectable.php

```diff
@@ -115,13 +115,11 @@ public function locate($name)
      *
      * @param string $name      the name of the resource to bind
      * @param mixed $value      the value of the resource
-     * @return Doctrine_Locator   this object
+     * @return void
      */
-    public function bind($name, $resource)
+    public function bind($name, $value)
     {
-        $this->_resources[$name] = $resource;
-        
-        return $this;    
+        $this->_resources[$name] = $value;
```

Comment:
Is it necessary to remove the `return $this`-Statement? This might bring a BC to users, relying on the result of `bind()`

Reply:
Reverted. The problem relies on the `class Doctrine_Table` which overrides the `bind()` method and does not return